crates/solidity/testing/sanctuary/src/main.rs

```diff
@@ -2,13 +2,18 @@ mod chains;
 mod datasets;
```

Comment:
nit suggestion: for file I/O in internal code, I suggest using our `Path::read_to_string()` and `Path::write_string()` utils, as it adds valuable error messages when something goes wrong.